docs/user/quickstart.rst

```diff
@@ -234,6 +234,45 @@ dictionary of data will automatically be form-encoded when the request is made::
       ...
     }
 
+You can also pass a list of tuples to the ``data`` argument. This is particularly
+useful when the form has multiple elements that use the same key::
+
+    >>> payload = (('key1', 'value1'), ('key1', 'value2'))
+    >>> r = requests.post('http://httpbin.org/post', data=payload)
+    >>> print(r.text)
+    {
+      ...
+      "form": {
+        "key1": [
+          "value1",
+          "value2"
+        ]
+      },
+      ...
+    }
+
+Value of each payload element can be a scalar or an iterable::
+
+    >>> payload = (('key1', ('value1', 1, (1, 2))), {'key2', ('value2', 2)})
```

Comment:
In that case,  this section goes away. I can just delete this section so only tuples section above remains. Does that sound good?